ert_shared/main.py

```diff
@@ -374,6 +401,12 @@ def get_ert_parser(parser=None):
             help="Disable monitoring.",
             default=False,
         )
+        cli_parser.add_argument(
```

Comment:
Would it make sense for the GUI also or should we wait? I think we wait.

Reply:
I think it makes sense! But I wanted it to also be exposed in the GUI in that case, so I think we wait.